lib/PostHog.php

```diff
@@ -4,6 +4,9 @@
 
 use Exception;
 
+ini_set('display_errors', 1);
+error_reporting(-1);
```

Comment:
what does this do? Set default to show errors or something?

Can the default be silent, and mention in the docs how to turn debug on?

Reply:
Oh this is cruft. I was testing this and couldn't quite get the levels to work